src/scripts/fix_workspace_info.py

```diff
@@ -0,0 +1,85 @@
+"""
+Fixes workspace info to do the following.
+1. Make sure the "narrative" metadata field contains an int that points to the narrative.
+2. Make sure the "narrative_nice_name" metadata field is correct.
+3. Make sure the "is_temporary" metadata field exists and is correct.
+4. Adds a count of the number of narrative cells.
+5. Does nothing at all if there's > 1 narrative in the workspace.
+Note that while this fetches the Narrative object, it doesn't modify it in any way.
+"""
+from biokbase.workspace.client import Workspace
+
+ws_url = "https://ci.kbase.us/services/workspace"
+
+def fix_workspace_info(ws_id, token, verbose=False):
+    """
+    ws_id = id of the workspace to update.
+    token = auth token of the user who owns the workspace.
+    """
+    assert(token is not None)
+    assert(str(ws_id).isdigit())
+
+    new_meta = dict()
+    ws = Workspace(url=ws_url, token=token)
+
+    if verbose:
+        print("Checking workspace {}".format(ws_id))
+
+    # test if there's exactly 1 Narrative object in the workspace
+    narr_obj_list = ws.list_objects({'ids': [ws_id]})
```

Comment:
This will list all objects in the ws, not just narratives. Filter by type would be helpful here.